### PR TITLE
:robot: Add encryption tests

### PR DESCRIPTION
**Pull request by Itxaka**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: tests/assets/autoinstall.yaml [hunk 1]
```diff
@@ -7,11 +7,9 @@ install:
...
-      users:
-        kairos:
-          passwd: "kairos"
-      hostname: kairos-{{ trunc 4 .Random }}
+hostname: metal-{{ trunc 4 .MachineID }}
+users:
+  - name: kairos
+    passwd: kairos
+    ssh_authorized_keys:
+      - github:Itxaka
```
Comment (mauromorales) [inline tests/assets/autoinstall.yaml:15]: Probably doesn't hurt to leave here, but just pointing it out in case it wasn't intentional

Comment (Itxaka) [reply tests/assets/autoinstall.yaml:15]: FML :rofl:

**Review by mauromorales — DISMISSED** [review]
lgtm

**Comment by Itxaka** [comment]
Already approved by @mauromorales only a fix for the autoinstall changes and squashing all commits into one. Merging!

**Pull request merged**